src/main/java/org/wso2/carbon/connector/utils/MessageBuilder.java

```diff
@@ -242,9 +248,20 @@ public MimeMessage build() throws MessagingException, IOException {
             MimeBodyPart body = new MimeBodyPart();
             setMessageContent(body);
             multipart.addBodyPart(body);
-            String[] attachFiles = attachments.split(",");
-            for (String filePath : attachFiles) {
-                addAttachment(multipart, filePath);
+            if (attachments.startsWith("[")) {
```

Comment:
shall we add a comment with the reason for this check

Reply:
Done